debian/clearwater-monit.service

```diff
@@ -4,7 +4,7 @@
 Description=Clearwater-monit service manager
 
 [Service]
-ExecStartPre=/usr/share/clearwater/bin/issue-alarm "systemd" "4500.6"
+ExecStartPre=[ -x /usr/share/clearwater/bin/issue-alarm ] && /usr/share/clearwater/bin/issue-alarm "systemd" "4500.6" || true
```

Comment:
Yes - but in a different package (so it shouldn't stop the clearwater-monit install)